pkg/controllers/securitypolicy_controller.go

```diff
@@ -53,6 +59,15 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	// Since securitypolicy service can only be activated from nsx 3.2.0 onwards,
+	// So need to check nsx version before starting securitypolicy Reconcile
+	if !r.Service.NSXClient.CheckNsxVersion() {
+		err := errors.New("check Nsx Version failed, would retry later")
+		log.Error(err, "check Nsx Version failed, would retry later")
```

Comment:
@dantingl  It will print error message twice
for this question, I feel it won't print twice, it just print for each CR once

Reply:
I mean it will print "check Nsx Version failed, would retry later" twice.